packages/kit/src/core/preview/index.js

```diff
@@ -120,6 +118,9 @@ export async function preview({
 				render_handler();
```

Comment:
It's unrelated to your change, but I wonder what is happening here. it looks like it'd just go in an infinite loop? maybe I'm misreading something

Reply:
I've been thrown by the structure of this file before. At this point we are outside the declaration of `render_handler`, and back into the callback to `get_server` (which is the request handler). This section is just deciding what paths to strip in what ways in order to handle the base and asset paths correctly. In this case it calls `render_handler` without calling `assets_handler` and `static_handler` first.

No loops, but I don't particularly like it.

Reply:
nevermind. I missed some of the braces when reading the code the first time around to see where the methods started and ended